### ISSUE TITLE
uses s32 instead of CutsceneData

### ISSUE THREAD

**Issue by ariahiro64** [sauraen/zcamedit#1]
Decomp uses the CutsceneData data type instead of s32.

**Comment by sauraen** [comment]
@Yanis42 is doing updates to fast64's cutscene handling to be in line with the decomp renamings, and also merging zcamedit into fast64. Once that is complete, this change will be effectively already done there, and this repo will be archived.
